### DISCUSSION TITLE
Bitclout is not a Startup

### DISCUSSION THREAD

**Discussion by ghost** [deso-protocol/dips#67] [Medium CIPs]
Bitclout project is an open source project. It will take its power from the developers. If it is not supported by enough developers, it has no chance of success. If Twitter has 1000 developers, we need 1001 developers to compete. Herein lies the advantage of being an open source project. Currently, most updates on the platform are made by the main developer team. And this team is acting like a low-budget start-up. I think the team should create a Developer team community without wasting time. We can't compete with Twitter with 10 Engineers. Bitclout is not a startup. I have some suggestions for this.

- We should encourage developers to Bitclout. For this, reward programs should be made.
- Voting for developers can be added on Bitclout.com. The developers with the most support in this vote should be rewarded. For the transparency of the voting, the voters must meet certain criteria (coin price, number of followers, member date, activity, etc.)
Thanks to this voting, users will also be able to participate in the administration. And that will make us more democratic.
- None of the 3rd party apps provide direct support to Bitclout. They don't create value without Bitclout. Thousands of dollars of user support is wasted. Instead, developers who add value to Bitclout through this voting system will be rewarded.
- Advertising should be done on LinkedIn platforms. Everyone who uses it knows it. A post containing the word "software" receives dozens of comments. There is a very high human resource there. If this audience supports Bitclout, we can do 1 year's work in 1 month.

**Comment by andyboyd** [comment]
✋  3rd party app developer here with flickapp. 

I disagree wholeheartedly with your point about 3rd party apps not providing direct support. We're running a node, so contributing to the network that way. We're actively working to bring quality creators onto the platform, and building a product to help them create high quality content, and motivate them to do so. We're also building iOS and Android libraries to make it easy for other 3rd party native apps to integrate with Identity (https://github.com/bitclout/identity-swift and https://github.com/bitclout/identity-kotlin). Yes, we stand to gain a lot from bitclout's success, but we're also putting in the hard yards to help grow it to get there.

It's not the core team's job to compete with twitter. They're building a protocol. IF someone wants to make a twitter clone on bitclout and directly compete, then they should hire those 1001 developers and do it. Personally, I'd far rather have an ecosystem where 100 small teams of 10 are building 100 exciting and diverse projects though.

**Reply by ghost** [reply]
I made a general determination above. Apart from that, I find Mobile application projects absolutely successful. Thanks to mobile application projects, the Bitclout protocol will be able to reach a large number of people. Millions of people in the world do not have computers but have smartphones. People of some countries met with mobile applications before computer technology came. In addition, there is a group far from technology in all countries of the world. This group can't even open a facebook account. The easiest way to reach these people is through mobile applications.

Sorry if I misunderstood. In my opinion, the projects that add the most value to Clout among 3rd party projects are mobile applications. What I mean is that getting as many developers as possible into the Bitclout protocol should be the main goal. More developers means more ideas, more features, and more free platforms. I worked on a blockchain-based social media platform 2-3 years ago. The difficulty we experienced while working was; The development team was weak. Because there is a new technology (blockchain) and you are competing with the biggest companies in the world by using this technology. Your engineer team is 10 people, but facebook's number of developers is 10*100. I was interested in local marketing. For example, we reached 50K unique users per day in a 3-month period in our country. But since the developer team was weak, we were constantly faced with new problems. There were also high cost server costs. As a result; Although we reach millions of people per month only in our country, we have not been successful. I think the only way we can be successful is to create a developer community. This developer community will continually improve the protocol. As the protocol develops, the quality of 3rd party projects will also increase. Therefore, at this stage, I think it is necessary for the project to reach the developers rather than reach the user by spending marketing studies. Bitclout is a real need in today's world. And when people hear about Bitclout, they will definitely join us.
